src/jit/compiler.cpp

```diff
@@ -7797,7 +7797,7 @@ void CompTimeSummaryInfo::Print(FILE* f)
         {
             fprintf(f,
                     "\n  'End phase slop' should be very small (if not, there's unattributed time): %9.3f Mcycles.\n",
```

Comment:
Not sure why this wasn't changed to correctly print the Mcycles, but if it's changed it should say "ms" instead of "Mcycles".

Reply:
The new compiler complains because the printf specifier wants a double for %9.3f
and the original provided argument was an __int64

Reply:
I can substitute ` m_total.m_parentPhaseEndSlop / 1000000.0 ` instead of fslop_ms

Reply:
Actually it looks like it should be printing the value of `fslop_ms `here, so I will change Mcycles to ms

Reply:
Actually I will change this to match the code at the other location and print both values (see line 7757)